src/Core/Domain/Customer/ValueObject/Email.php

```diff
@@ -26,106 +26,14 @@
 
 namespace PrestaShop\PrestaShop\Core\Domain\Customer\ValueObject;
 
-use PrestaShop\PrestaShop\Core\Domain\Customer\Exception\CustomerConstraintException;
-
 /**
- * Stores customer's email address
+ * @deprecated since version 1.7.6.4 use PrestaShop\PrestaShop\Core\Domain\ValueObject\Email instead
```

Comment:
@atomiix Sure about that ?

Reply:
@matks yep, this class is not used. Also, this PR is a cherry-picking of another PR on the 1.7.7.x branch that makes email validation coherent.